stories/Chrome.stories.js

```diff
@@ -21,5 +21,20 @@ storiesOf('LayoutChrome', module).add('On state', () => ({
 
 storiesOf('LayoutChrome', module).add('On state with text', () => ({
```

Comment:
This story has stopped working due to Vuex. Prioritizing manual testing and functional tests to getting storybook to work atm.